src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -226,7 +247,9 @@ def test_upper_bound_fortran_2(monkeypatch):
 
 def test_upper_bound_inner(monkeypatch):
     ''' Check that we get the correct Fortran generated if a loop's upper
-    bound is "inner". '''
+    bound is "inner".
```

Comment:
I would suggest `'inner'` for consistency with similar docstrings.

Reply:
done